modules/uac/auth.c

```diff
@@ -120,7 +121,9 @@ int apply_cseq_op(struct sip_msg *msg,int val)
 		return -1;
 	}
 
-	cseq_no=cseq_no+val;
+	if (increment_cseq) {
+		cseq_no=cseq_no+val;
```

Comment:
I'd suggest adding some spaces here in the vicinity while you're touching the code.
```
cseq_no = cseq_no + val;
```
or
```
cseq_no += val;
```
.. is so much more readable.